lib/matplotlib/contour.py

```diff
@@ -1749,7 +1752,7 @@ def _initialize_x_y(self, z):
             however introduce rendering artifacts at chunk boundaries depending
             on the backend, the *antialiased* flag and value of *alpha*.
 
-        linewidths : float or sequence of float, default: :rc:`lines.linewidth`
+        linewidths : float or sequence of float, default: :rc:`contour.linewidths`
```

Comment:
You can use
```suggestion
        linewidths : float or array-like, default: :rc:`contour.linewidths`
```

which is also more in line with our current documentation standard (https://matplotlib.org/devdocs/devel/documenting_mpl.html#parameter-type-descriptions).

Reply:
Excellent! Thanks.